change/@internal-react-composites-0d94c318-0522-4ac7-a3b4-dc495fb5e32c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add TeamsCall and TeamsCallAgent to stateful - Add new beta feature flag \"teams-call\" - Extract common logic CallAgentCommon and CallCommon",
+  "packageName": "@internal/react-composites",
```

Comment:
looks like there's no longer changes to this package - remove (or change to `none` if the update to the tests folder triggers this)